packages/ui-components/src/Input/TextTypeInput.tsx

```diff
@@ -57,18 +43,18 @@ export const BaseTextInput: React.FC<InternalTextProps> = props => {
     type = "text",
     prefixElement,
     ariaLabelledBy,
-    existingPassword = false,
-    forgotPasswordLinkElement,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    inline,
```

Comment:
De-structured because we shouldn't be passing this to `inputProps`, as it causes a noisy error since we're trying to pass an unrecognized prop to the DOM input element. Open to other suggestions of implementation - seems the alternative is to destructure all `inputProps`, which runs into the issue we had before of having to pass `onChange`,`name`, `onFocus`, etc.